modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-fragment/visualizationModes.spec.ts

```diff
@@ -30,8 +30,6 @@ test.beforeEach(async ({dataSetManagerApiHelpers}) => {
 	dataSetERC = getRandomString();
 
 	await dataSetManagerApiHelpers.createDataSet({
-		additionalAPIURLParameters:
-			'&nestedFields=dataSetToDataSetTableSections',
```

Comment:
Are you sure we don't need this? I remember adding this because without it `SAMPLE_OBJECT_FIELD` would not be an object. Do we not need it?

Reply:
The `&nestedFields=dataSetToDataSetTableSections` parameter is the part that we were trying to fix with this PR. Adding it manually made the test pass without the fragmentRenderer calculating correctly the `nestedFields`.

Reply:
Ooh I see. Makes sense!